sdks/java/core/src/main/java/org/apache/beam/sdk/runners/inprocess/InProcessBundleFactory.java

```diff
@@ -48,9 +48,7 @@ private InProcessBundleFactory() {}
 
   @Override
   public <T> UncommittedBundle<T> createBundle(CommittedBundle<?> input, PCollection<T> output) {
-    return input.isKeyed()
-        ? InProcessBundle.keyed(output, input.getKey())
-        : InProcessBundle.unkeyed(output);
+    return InProcessBundle.keyed(output, input.getKey());
```

Comment:
s/keyed/create

removed unkeyed, and just use null explicitly. As the key is user-specified, we must handle null properly already.